util/chunk/disk.go

```diff
@@ -44,9 +45,9 @@ var bufReaderPool = sync.Pool{
 	New: func() interface{} { return bufio.NewReaderSize(nil, readBufSize) },
 }
 
-var tmpDir = filepath.Join(os.TempDir(), "tidb-server-"+filepath.Base(os.Args[0]))
+func initTempDir() string {
+	tmpDir := filepath.Join(config.GetGlobalConfig().TempPath, "tidb-server-"+filepath.Base(os.Args[0]))
```

Comment:
I think the sub directory should also be determined by config.
```suggestion
	tmpDir := config.GetGlobalConfig().TempPath
```